e2e/cypress/integration/api/iam/policies_api.spec.ts

```diff
@@ -29,78 +54,210 @@ describe('policies API', () => {
-        ...defaultAdminReq,
-        method: 'POST',
-        failOnStatusCode: false,
-        body: {
-          id: `${cypressPrefix}-policy-${now}`,
-          name: `${cypressPrefix} policy ${now}`,
-          members: ['team:local:test'],
-          projects: [project1.id]
-        }
-      }).then((response) => {
-       expect(response.status).to.equal(400);
+    describe('POST /apis/iam/v2/policies', () => {
+
+      it('returns 200 when all valid inputs are provided', () => {
+        cy.request({
+          ...defaultAdminReq,
+          method: 'POST',
+          failOnStatusCode: false,
+          body: {
+            id: policyID,
+            name: policyName,
+            members: ['team:local:test'],
+            statements: [
+              {
+                effect: 'ALLOW',
+                actions: ['test:svc:someaction', 'test:svc:otheraction'],
+                projects: [project1.id]
+              }
+            ],
+            projects: [project1.id]
+          }
+        }).then((response) => {
+          expect(response.status).to.equal(200);
+          expect(response.body.policy.id).to.equal(policyID);
+          expect(response.body.policy.name).to.equal(policyName);
+          expect(response.body.policy.members).to.deep.equal(['team:local:test']);
+          expect(response.body.policy.statements).to.deep.equal([{
+            effect: 'ALLOW',
+            actions: ['test:svc:someaction', 'test:svc:otheraction'],
+            resources: ['*'],
+            role: '',
+            projects: [project1.id]
+          }]);
+          expect(response.body.policy.projects).to.deep.equal([project1.id]);
+        });
       });
-    });
-  });
 
-  describe('PUT /apis/iam/v2/policies', () => {
-    beforeEach(() => {
-      cy.cleanupV2IAMObjectsByIDPrefixes(cypressPrefix, ['policies']);
+      it('returns 400 when there are no statements', () => {
+        cy.request({
+          ...defaultAdminReq,
+          method: 'POST',
+          failOnStatusCode: false,
+          body: {
+            id: `${cypressPrefix}-policy-${now}`,
+            name: `${cypressPrefix} policy ${now}`,
+            members: ['team:local:test'],
+            projects: [project1.id]
+          }
+        }).then((response) => {
+          expect(response.status).to.equal(400);
+        });
+      });
     });
 
-    afterEach(() => {
-      cy.cleanupV2IAMObjectsByIDPrefixes(cypressPrefix, ['policies']);
+    describe('GET', () => {
+
+      it('returns 200 on successful fetch (/apis/iam/v2/policies)', () => {
+        cy.request({
+          ...defaultAdminReq,
+          method: 'GET',
+          url: '/apis/iam/v2/policies'
+        }).then((response) => {
+          expect(response.status).to.equal(200);
+          expect(response.body.policies.length).to.equal(chefManagedPolicyIDs.length + 1);
+
+          const policyIDs = response.body.policies.map((pol: Policy) => pol.id);
+
+          chefManagedPolicyIDs.forEach((id: string) => {
```

Comment:
Type is inferred so explicit type is not needed:
```suggestion
          chefManagedPolicyIDs.forEach(id => {
```